sdks/python/examples/fanout/stream.py

```diff
@@ -21,9 +22,8 @@ async def main() -> None:
     # This key gets propagated to all child workflows
     # and can have an arbitrary property name.
 
-    hatchet.admin.run_workflow(
-        "Parent",
-        {"n": 2},
+    parent_wf.run(
```

Comment:
https://github.com/hatchet-dev/hatchet/pull/1316/files sdks/typescript/src/v1/workflow.ts

I'm playing with the idea of simplifying this further by exposing 2 endpoints

1. enqueue which behaves likes the current run -- returns the ref where you can get the id or await a result

2. run which awaits the result directly

Reply:
I think this is pretty reasonable - seems like a common pattern to want to get the results of child runs before returning, and this feels similar to `FireAndForget` / `FireAndWait`. WDYT?